docs/querying/sql-translation.md

```diff
@@ -71,7 +71,7 @@ EXPLAIN PLAN statements return:
 - a `RESOURCES` column that describes the resources used in the query
 - an `ATTRIBUTES` column that describes the attributes of the query, including:
   - `statementType`: the SQL statement type
-  - `targetDataSource`: the target datasource in an INSERT or REPLACE statement
+  - `targetDataSource`: a JSON object representing the target datasource in an INSERT or REPLACE statement.
   - `partitionedBy`: the time-based partitioning granularity in an INSERT or REPLACE statement
```

Comment:
For 29.0.0, could you also please update `targetDataSource` format in this doc to the JSON object in these example `EXPLAIN PLAN` outputs [here](https://github.com/apache/druid/blob/master/docs/querying/sql-translation.md?plain=1#L447) and [here](https://github.com/apache/druid/blob/master/docs/querying/sql-translation.md?plain=1#L668)?

Reply:
Changed